src/services/application.js

```diff
@@ -153,6 +153,7 @@ const ApplicationService = function() {
 
   this.changeLanguage = function(language){
     changeLanguage(language);
+    ApplicationState.lng = language; // DEPRECATED: this property will be removed on v4 version. Leave for backward compatibility
```

Comment:
Whenever possible make use of the [@deprecated](https://jsdoc.app/tags-deprecated.html) tag so that your IDE knows it too (syntax higliting, code linting, ..)

In the case of small annotations you can also use the inline version:

```js
/** @deprecated since v3.7 */
foo: "bar"
```

instead of:

```js
/**
 * @deprecated since v3.7
 */
foo: "bar"
```